### PR TITLE
Analytics

### PR DESCRIPTION
**Pull request by sinan**
- adds the initial analytic app under `/client` at `<host>/Analytics` route
- adds the first es6 app 
- adds the eslint config

to have the menu item in the account dropdown, you need to have a cookie set named `countly-enabled`

### CODE REVIEW

File: client/analytics/bant.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "analytics",
+  "main": "./lib/index.coffee",
```
Comment (usirin) [inline client/analytics/bant.json]: `index.js`

File: client/analytics/lib/routehandler.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import kd from 'kd'
+import lazyrouter from 'app/lazyrouter'
+
+export default function () {
+  return lazyrouter.bind('analytics', (type, info, state, path, ctx) => {
+    return kd.singletons.appManager.open('Analytics')
+  })
```
Comment (usirin) [inline client/analytics/lib/routehandler.js]: do we need the explicit `return`s here?

File: client/package.json [hunk 3]
```diff
@@ -123,6 +123,10 @@
     "coffee-script": "1.12.4",
     "copy-webpack-plugin": "4.0.1",
     "css-loader": "0.26.1",
+    "eslint": "^3.18.0",
+    "eslint-config-standard": "^7.1.0",
+    "eslint-plugin-promise": "^3.5.0",
+    "eslint-plugin-standard": "^2.1.1",
```
Comment (usirin) [inline client/package.json]: `^` will fail the builds

File: config/generateKonfig.coffee [hunk 4]
```diff
@@ -127,6 +127,7 @@ module.exports = (options, credentials) ->
 
     terraformerSecretKey: credentials.terraformer.secretKey
 
+  credentials.countly.host = 'http://dev.koding.com:32768/countly' if options.countlyPath
```
Comment (usirin) [inline config/generateKonfig.coffee]: should we read this from `hostname` or another appropriate `configure` option?

Comment (sinan) [inline config/generateKonfig.coffee]: this will change, probably how you suggest. cc/ @cihangir

Comment (cihangir) [inline config/generateKonfig.coffee]: this is done [here](https://github.com/koding/vault/pull/68) we can remove this line

File: go/build.sh [hunk 5]
```diff
@@ -7,7 +7,7 @@ export GOBIN=${GOBIN:-${GOPATH}/bin}
 export KODING_REPO=$(git rev-parse --show-toplevel)
 export KODING_GIT_VERSION=$(git rev-parse --short HEAD || cat ./VERSION || cat ../VERSION || cat ../../../VERSION || echo "0")
 export KODING_VERSION=${KODING_VERSION:-$KODING_GIT_VERSION}
-export KODING_LDFLAGS="-X koding/artifact.VERSION=${KODING_VERSION} -X main.GitCommit=${KODING_VERSION}"
+export KODING_LDFLAGS="-X koding/artifact.VERSION=${KODING_VERSION} -X main.GitCommit=${KODING_VERSION} -s"
```
Comment (cihangir) [inline go/build.sh]: cc/ @koding/godevs

**Review by cihangir — APPROVED** [review]

**Pull request merged**